dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/ExecutionStatus.java

```diff
@@ -90,7 +90,7 @@ public boolean typeIsSuccess() {
      * @return status
      */
     public boolean typeIsFailure() {
-        return this == FAILURE || this == NEED_FAULT_TOLERANCE || this == KILL;
+        return this == FAILURE || this == NEED_FAULT_TOLERANCE;
     }
```

Comment:
What worries me is that modifying this will cause abnormal business logic judgments elsewhere

Reply:
i checked the same method in release-1.3.5,it just has two,and i also check the usage of this method,dev branch has one more usage,and i also change it.please see the pictures